.codeclimate.yml

```diff
@@ -1,27 +1,54 @@
-engines:
-  brakeman:
-    enabled: true
-  bundler-audit:
-    enabled: true
-  csslint:
-    enabled: true
-  duplication:
-    enabled: true
-    exclude_paths:
-      - spec/**/*
-  flog:
-    enabled: true
-    exclude_paths:
-      - db/**/*
-      - config/**/*
-      - spec/**/*
-  rubocop:
-    enabled: true
-    exclude_paths:
-      - db/**/*
-    channel: rubocop-0-49
-ratings:
-  paths:
-  - app/**
-  - lib/**
-  - "**.rb"
+version: 2
+jobs:
+  build:
+    environment:
+      # You must replace the CC_TEST_REPORTER_ID with the one of your project
+      CC_TEST_REPORTER_ID: ff537d301db6b5db99dc8e27242638e13095a7bae63f7a232dac5d0827c797c6
```

Comment:
can this be read from an env var?

Reply:
It's not sensible data according to https://docs.codeclimate.com/docs/finding-your-test-coverage-token#section-regenerating-a-repos-test-reporter-id

Also you can't add env vars the codeclimate environment as far as i know.
```
Should I keep my Test Reporter ID secret?
Your repo's test coverage ID is only used to identify your repo when submitting test coverage payloads. It can't be used to access any of your data.```